python/cudnn/gemm/frost/compiler.py

```
@@ -1081,24 +1080,30 @@ def _bs_smem_desc_params(is_mn_major, mn_extent, name):
     sf_k4 = sf_k // 4  # 4 SF-K per utccp atom
     nb_m = cta_m // 128
     nb_n = cta_n // 128
+    mma_nb_m = cfg.mma_inst_m // 128
+    mma_nb_n = cfg.mma_inst_n // 128
+    sfa_nb_m = mma_nb_m * cfg.num_mma_m
 
     _REGISTERS_PER_ATOM = 4  # cols per 128×4 utccp atom
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)